client/jetpack-cloud/sections/pricing/jpcom-masterbar/style.scss

```diff
@@ -1334,5 +1342,9 @@
 			z-index: 9999999;
 		}
 	}
+	//logo is really close to the top of the page on mobile
+	.jetpack-logo {
+		margin-top: 25px;
```

Comment:
Sure, I can do that. In that style.scss there's a bunch of other `px` so either one fits but can patch since most of the other margin-tops are in rem though